src/.nuget/dir.targets

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <!--
+    Finds symbol files and injects them into the package build.
+  -->
+  <Target Name="GetSymbolPackageFiles" BeforeTargets="GetPackageFiles">
+    <ItemGroup Condition="'$(SymbolFileExtension)' != ''">
+      <AdditionalLibPackageExcludes Include="%2A%2A\%2A$(SymbolFileExtension)"/>
+    </ItemGroup>
+
+    <ItemGroup>
+      <AdditionalLibPackageExcludes Include="@(LongNameFile -> '%(TargetPath)')" />
+    </ItemGroup>
+
+    <ItemGroup>
+      <NativeWithSymbolFile Include="@(NativeBinary);
+                                     @(ArchitectureSpecificNativeFile);
```

Comment:
What's the difference between a "NativeBinary" item and an "ArchitectureSpecificNativeFile"?